packages/capacitor-plugin/README.md

```diff
@@ -597,9 +597,9 @@ Callback for receiving chunks read from a file, or error if something went wrong
 | **`ExternalStorage`** | <code>'EXTERNAL_STORAGE'</code> | The external storage directory. On iOS it will use the Documents directory. On Android it's the primary shared/external storage directory. It's not accesible on Android 10 unless the app enables legacy External Storage by adding `android:requestLegacyExternalStorage="true"` in the `application` tag in the `AndroidManifest.xml`. It's not accesible on Android 11 or newer.                                                                                                      | 1.0.0 |
-| **`ExternalCache`**   | <code>'EXTERNAL_CACHE'</code>   | The external cache directory. Android ONly On Android it's the primary shared/external cache. It's not accesible on Android 10 unless the app enables legacy External Storage by adding `android:requestLegacyExternalStorage="true"` in the `application` tag in the `AndroidManifest.xml`. It's not accesible on Android 11 or newer.                                                                                                                                                   | 7.1.0 |
-| **`LibraryNoCloud`**  | <code>'LIBRARY_NO_CLOUD'</code> | iOS only It maps to Library/NoCloud directory Files will be deleted when the application is uninstalled.                                                                                                                                                                                                                                                                                                                                                                                  | 7.1.0 |
-| **`Temporary`**       | <code>'TEMPORARY'</code>        | iOS only The tmp/ directory. Files will be deleted when the application is uninstalled.                                                                                                                                                                                                                                                                                                                                                                                                   | 7.1.0 |
+| **`ExternalCache`**   | <code>'EXTERNAL_CACHE'</code>   | The external cache directory. On iOS it will use the Documents directory. On Android it's the primary shared/external cache. It's not accesible on Android 10 unless the app enables legacy External Storage by adding `android:requestLegacyExternalStorage="true"` in the `application` tag in the `AndroidManifest.xml`. It's not accesible on Android 11 or newer.                                                                                                                    | 7.1.0 |
```

Comment:
```suggestion
| **`ExternalCache`**   | <code>'EXTERNAL_CACHE'</code>   | The external cache directory. On iOS it will use the Documents directory. On Android it's the primary shared/external cache. It's not accessible on Android 10 unless the app enables legacy External Storage by adding `android:requestLegacyExternalStorage="true"` in the `application` tag in the `AndroidManifest.xml`. It's not accesible on Android 11 or newer.                                                                                                                    | 7.1.0 |
```